feature-group-definitions/font-optical-sizing.yml

```diff
@@ -1,4 +1,5 @@
 name: font-optical-sizing
+description: The `font-optical-sizing` CSS property sets whether text rendering is optimized for viewing at different sizes.
```

Comment:
I guess that `font-optical-sizing: none` is the new feature here. But not calling that out seems OK.